src/server/user.jsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react';
```

Comment:
This file has chaged to `src/server/pages/user-page.jsx` (and has somewhat different logic) in master. I'm not sure if this PR needs to be rebased, or if it already was but it was performed somewhat incorrectly?

Reply:
Oh, looks like I have both for some reason.